openedx/core/djangoapps/video_pipeline/models.py

```diff
@@ -80,9 +92,21 @@ def get_service_user(self):
 @python_2_unicode_compatible
 class VideoUploadsEnabledByDefault(ConfigurationModel):
     """
-    Enables video uploads enabled By default feature across the platform.
+    Enables video uploads across the platform.
 
     .. no_pii:
+
+    .. toggle_name: VideoUploadsEnabledByDefault.enabled_for_all_courses
+    .. toggle_implementation: ConfigurationModel
+    .. toggle_default: False
+    .. toggle_description: Allow video uploads for all courses of the platform. This
+      enables the "Video Uploads" menu in the CMS.
+    .. toggle_category: video_pipeline
+    .. toggle_use_cases:  open_edx
+    .. toggle_creation_date: 2017-11-10
+    .. toggle_target_removal_date: None
+    .. toggle_warnings: None
```

Comment:
Remove?

Reply:
:ok_hand: I removed all instances of `.. toggle_warnings: None`.